File: Backend/ExamManagementSystem-solution/ExamManagementSystem/Models/ExamMaster.cs
using System;
using System.Collections.Generic;

namespace ExamManagementSystem.Models;

public partial class ExamMaster
{
    public int MasterId { get; set; }

    public int StudentId { get; set; }

    public int ExamYear { get; set; }

    public int TotalMark { get; set; }

    public string PassOrFail { get; set; } = null!;

    public DateTime CreateTime { get; set; }

    public virtual ICollection<ExamDtl> ExamDtls { get; set; } = new List<ExamDtl>();

    public virtual StudentMst Student { get; set; } = null!;
}

File: Frontend/ExamManagementSystem/src/app/models/exam-master.ts
import { ExamDetail } from "./exam-detail";

export class ExamMaster {

  masterId: number = 0;

  studentId: number = 0;

  studentName: string = '';

  mail: string = '';

  examYear: number = 0;

  totalMark: number = 0;

  passOrFail: string = '';

  createTime?: Date;

  examDtls: ExamDetail[] = [];
}
